lib/modernizr-build.js

```diff
@@ -0,0 +1,26 @@
+const modernizr = require('modernizr')
+const fs = require('fs-extra')
+const path = require('path')
+const paths = require('../config/paths.json')
+
+/**
+ * Metalsmith plugin to create a custom modernizr build
+ *
+ * @return {Function}
+ */
+
+const plugin = (opts) => {
+  opts = opts || {}
+  let buildConfig = require(opts.config)
+  return function (files, metalsmith, done) {
+    modernizr.build(buildConfig, (content) => {
+      fs.ensureDir(path.join(paths.public, opts.destination), err => {
```

Comment:
ensures that the directory exists, if the directory structure does not exist, it is created. same as `mkdir -p`. maybe it's worth adding a comment above.

Reply:
added